leaderboard_v1.html

```diff
@@ -0,0 +1,324 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-NRZJLJCSH6"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+        gtag("js", new Date());
+
+        gtag("config", "G-NRZJLJCSH6");
+    </script>
+    <script src="assets/lib/chart.umd.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/chartjs-plugin-autocolors"></script>
+    <script type="text/javascript">
+        window.PlotlyConfig = { MathJaxConfig: "local" };
+    </script>
+    <script type="text/javascript" src="treemap.js"></script>
+
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description"
+        content="Explore The Berkeley Function Calling Leaderboard (also called The Berkeley Tool Calling Leaderboard) to see the LLM's ability to call functions (aka tools) accurately." />
+    <link rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro" />
+    <link rel="stylesheet" href="assets/css/api-explorer.css" />
+    <link rel="stylesheet" href="assets/css/common-styles.css" />
+    <link rel="stylesheet" href="assets/css/Highlight-Clean-leaderboard.css" />
+    <link rel="stylesheet" href="assets/css/leaderboard.css" />
+    <link rel="stylesheet" href="assets/css/leaderboard_v1.css" />
+    <link rel="stylesheet" href="assets/css/treemap.css" />
+    <link rel="stylesheet" href="assets/css/contact.css" />
+    <link rel="stylesheet" href="assets/css/styles.css" />
+    <title>
+        Berkeley Function Calling Leaderboard (aka Berkeley Tool Calling Leaderboard)
```

Comment:
Like we discussed this might change, but for future reference, we shouldn't have 2 pages with the same "title" ?

Reply:
addressed